test/plugin_functional/plugins/usage_collection/server/routes.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { HttpServiceSetup } from 'kibana/server';
+import { UsageCounter } from '../../../../../src/plugins/usage_collection/server';
+
+export function registerRoutes(http: HttpServiceSetup, usageCounter: UsageCounter) {
+  const router = http.createRouter();
+  router.post(
+    {
+      path: '/api/usage_collection_test_plugin/',
```

Comment:
NIT: 
- remove trailing `/`
- Why `post` instead of `get` is there is no payload?